launchlib/config_test.go

```diff
@@ -125,7 +127,7 @@ subProcesses:
 				VersionedConfig: VersionedConfig{
 					Version: 1,
 				},
-				ServiceName: "",
```

Comment:
Confirmed that sls-packaging actually does fill in sensible values here (not hard-coded to primary, but the actual service name)